package-lock.json

```diff
@@ -17369,7 +17369,7 @@
         },
         "are-we-there-yet": {
           "version": "1.1.4",
-          "resolved": false,
+          "resolved": "",
```

Comment:
@aaemnnosttv I think you might be using a different local NPM version so that all these weird changes are made. It was happening for me as well at some point until I updated it.

My versions are:
* Node: v10.16.3
* NPM: 6.10.3

Reply:
I'm using the same version of node, but `npm` 6.9.0, perhaps that's it?

Once we hit v1 we can automate release so these kinds of things won't be an issue anymore, otherwise I'd say we should create a docker container/env for this.